src/style/app.module.css

```diff
@@ -30,6 +30,7 @@
   --tablerow-bg-color: #eff1f7;
   --row-background: var(--bs-white, white);
   --font-size-header: 16px;
+  --bg-blue: #afffe8;
```

Comment:
This isn't blue, it's green.  It's not in keeping with the color scheme. Use this to see.
- https://www.w3schools.com/colors/colors_picker.asp